FrostyCmd/Profiles/BF2042.json

```diff
@@ -0,0 +1,36 @@
+{
+  "Name": "BF2042",
+  "DisplayName": "Battlefield™ 2042",
+  "InternalName": "bf2042",
+  "Deobfuscator": "Signature",
+  "TypeInfoSignature": "",
+  "DataVersion": 20211119,
+  "FrostbiteVersion": "2021-1.1",

```

Comment:
fb versions need to be in 20XX.X.X format even if the actual fb version format changed